### ISSUE TITLE
Do you have plan to improve the test accuracy?

### ISSUE THREAD

**Issue by GitGhidorah** [kamalkraj/Vision-Transformer#1]
Hello!!
Im beginner.
Do you have plan to improve the test accuracy?

I tried to run for 300 epochs. The result is below.
   epoch 300: train loss 0.10614. train accuracy 0.96536
   epoch 300: test loss 5.57218. test accuracy 0.48910
Thank you.

**Comment by kamalkraj** [comment]
Hi @GitGhidorah,
Try Different hyperparameters, The hyperparameter in the notebook is just a sample not the optimal.

**Comment by kamalkraj** [comment]
HI @GitGhidorah,
Try model configs from https://github.com/google-research/vision_transformer/blob/master/vit_jax/configs.py
